llama_stack/apis/common/errors.py

```diff
@@ -50,3 +50,11 @@ def __init__(self, toolgroup_name: str) -> None:
             f"Tool group '{toolgroup_name}' not found. Use client.toolgroups.list() to list available tool groups."
         )
         super().__init__(message)
+
+
+class MissingTrainingConfigError(ValueError):
+    """raise when Llama Stack is missing configuration for training"""
+
+    def __init__(self, config_name: str) -> None:
+        message = f"'{config_name}' is required for training with this provider"
```

Comment:
what does "this provider" mean when a user sees this message?

Reply:
i couldn't find a way to identify the exact provider used when the exception is raised, but if there is a way to do it that would definitely be my preference

Reply:
I took a stab at a more precise solution, but let me know if you think there is a more elegant way to implement this